src/helpers/game_utils.ts

```diff
@@ -185,26 +190,8 @@ async function selectRandomSong(guildPreference: GuildPreference): Promise<Queri
         return null;
     }
 
-    let attempts = 0;
-    while (true) {
-        // this case should rarely happen assuming our song cache is relatively up to date
-        if (attempts > 5) {
-            logger.error(`Failed to select a random song: guildPref = ${JSON.stringify(guildPreference)}`);
-            return null;
-        }
-        const random = queriedSongList[Math.floor(Math.random() * queriedSongList.length)];
-
-        if (isDebugMode() && skipSongPlay()) {
-            return random;
-        }
-        const songLocation = `${SONG_CACHE_DIR}/${random.youtubeLink}.mp3`;
-        if (!fs.existsSync(songLocation)) {
-            logger.error(`Song not cached: ${songLocation}`);
-            attempts++;
-            continue;
-        }
-        return random;
-    }
+    const random = queriedSongList[Math.floor(Math.random() * queriedSongList.length)];
```

Comment:
Could just return instead of assigning a variable, I don't think it's being used anywhere else